docs/farming-&-staking/farming/_category_.json

```diff
@@ -1,8 +1,8 @@
 {
     "label": "Farming",
-    "position": 3,
+    "position": 1,
```

Comment:
It's a bit weird that Wallets come after farming, I'd suggest keeping the order:

Wallets, Farming, Operators & Nominators, Timekeeping